src/components/notificationPopUp/NotificationPopUp.tsx

```diff
@@ -13,14 +13,15 @@ export interface NotificationPopUpProps {
   hide: () => void;
   primaryButton: {
     label: string;
-    icon?: JSX.Element;
     handleClick: () => any;
+    icon?: JSX.Element;
```

Comment:
Might be out-of-scope for this pull request, but, I think we should stop passing things like this as an `JSX.Element`.

Reason: we can now put **any element** in this prop, creating the possibility for many issues.

For example: the `description` prop here, is a `JSX.Element` as well, and we're putting it in a `<Paragraph>`. However, we have implementations in which we're pushing `<Table>` elements into this `<Paragraph>`, which is not allowed.